app/components/Button/styles.css

```diff
@@ -6,15 +6,20 @@
   justify-content: center;
   align-items: center;
   transition: background 0.2s, color 0.2s;
+  outline: none;
 }
 
 .default {
-  background: #303B4B;
+  background: #303b4b;
   color: white;
 }
 
+.rounded {
+  border-radius: 9999px;
```

Comment:
Lol wut? I think 100px would be enough and accomplish the same thing :P

Reply:
Yes, assuming we don't have very large buttons, which we probably can do. I will change it.